curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9343769e8f85c1e17e05.md

```diff
@@ -14,13 +14,18 @@ Create a `div` element with the class of `task`. Utilize template strings to set
 You should create a `div` element with the class `task`.
 
 ```js
-assert.match(code, /<div\s+class\s*=\s*('|")task\1/)
+const classFirst =  /<div\s+class\s*=\s*('|")task\1/.test(code);
+const idFirst =  /<div\s+id\s*=\s*('|")task\1\s+class\s*=\s*('|")task\2/.test(code);
+assert.isTrue(classFirst || idFirst);
```

Comment:
```suggestion
const classFirst = `<div\s+class\s*=\s*('|")task\1`;
const idFirst = `<div\s+id\s*=\s*('|")task\1\s+class\s*=\s*('|")task\2`
assert.match(code, new Regex(`${classFirst}|${idFirst}`));
```
like this maybe?

Reply:
also, why there is `id=task` here? shouldn't the id be `${id}`?

Reply:
if you don't like it it's not necessary to use an OR inside the regex, but the `id=task` needs to be changed

Reply:
you still have id=task in the regex